Dockerfile

```diff
@@ -0,0 +1,25 @@
+# kovri - I2P but in CPP
+# Copyright (c) 2015-2017, The Kovri I2P Router Project (see LICENSE.md)
+# docker build -t geti2p/kovri .
+# KOVRI_PORT=42085 && sudo docker run -p $KOVRI_PORT --env KOVRI_PORT=$KOVRI_PORT geti2p/kovri
+
+FROM alpine:3.5
+RUN apk add --update --no-cache -t .kovri-dev g++ make cmake binutils boost-dev libressl-dev
+ENV KOVRI_HOST 0.0.0.0
+ENV KOVRI_PORT 24085
+ADD . kovri/
```

Comment:
Neither am i :) but i use it a lot. Major point is self-consistency : no additional step. It's supposed to describe the entire process. By using "ADD", somehow you are making a step less explicit. Also easier to deploy to dockerhub : no need to maintain a local copy of the source there.